### PR TITLE
Migrate from preact to react 

### PR DESCRIPTION
**Pull request by Sharqiewicz**
With all the packages in the portal updated in the last 2 weeks, the migration was really straightforward.

What have I done?

1. Replace the Preact-related packages with React and React-dom
2. Remove all properties specific to Preact
3. Migrate event system to React synthetic event system
4. Remove deprecated babel configuration for tests in the project 
5. Remove all outdated and unused files and configurations

to do:
- [x] test all the functionalities (99%)
- [x] update readme 
- [x] review all todo (in the code)

tested:

Staking
- [x] stake
- [x] unstake
- [ ] unlock 
- [x] rewards update
- [x] rewards claim 

Forex AMM
- [x] Swap 
- [x] Swap pools Deposit/Withdraw
- [x] Backstop pools Deposit/Withdraw 

Spacewalk
- [x] Bridge
- [x] Transactions

Wallet connection
- [x] Connect
- [x] Disconnect

### CODE REVIEW

File: src/components/Form/From/NumericInput/helpers.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { ChangeEvent, ClipboardEvent, KeyboardEvent } from 'react';
```
Comment (TorstenStueber) [inline src/components/Form/From/NumericInput/helpers.ts:1]: I don't think that we use `ClipboardEvent` in this file.

File: src/components/Form/From/NumericInput/index.tsx [hunk 2]
```diff
@@ -32,28 +33,28 @@ export const NumericInput = ({
...
-    <div className="flex w-full justify-between">
-      <div className="font-outfit flex-grow text-4xl text-black">
+    <div className="flex justify-between w-full">
+      <div className="flex-grow text-4xl text-black font-outfit">
         <Input
           {...register}
-          autocomplete="off"
-          autocorrect="off"
-          autocapitalize="none"
+          autoComplete="off"
```
Comment (TorstenStueber) [inline src/components/Form/From/NumericInput/index.tsx:40]: Where these attributes wrong before and did not have any effect?

File: src/components/Layout/Nav/NavItem/index.tsx [hunk 3]
```diff
@@ -23,13 +23,16 @@ export const NavItem = ({
   if (hidden) return null;
   const isExternal = isExternalLink(link);
 
+  if (isLottieOptions(prefix) || isLottieOptions(title)) return <></>;
```
Comment (TorstenStueber) [inline src/components/Layout/Nav/NavItem/index.tsx:26]: Why this change? This seems to apply to the Spacewalk button. Wouldn't that then skip rendering for this button?

File: src/components/nabla/Pools/Backstop/AddLiquidity/index.tsx [hunk 4]
```diff
@@ -115,7 +115,7 @@ const AddLiquidity = ({ data, onClose }: AddLiquidityProps): JSX.Element | null
                 color="secondary"
                 className="mt-2 w-full"
                 type="button"
-                disable={mutation.isLoading}
+                disabled={mutation.isLoading}
```
Comment (TorstenStueber) [inline src/components/nabla/Pools/Backstop/AddLiquidity/index.tsx:118]: This was also an ineffective attribute?

File: src/main.tsx [hunk 5]
```diff
@@ -11,35 +10,37 @@ import { tenantTheme, ThemeName } from './models/Theme';
...
+                  </SharedProvider>
+                </NodeInfoProvider>
+              );
+            }}
+          </GlobalStateContext.Consumer>
+        </GlobalStateProvider>
+      </BrowserRouter>
+      <ReactQueryDevtools initialIsOpen={false} position="bottom-right" />
+    </QueryClientProvider>
+    ,
```
Comment (TorstenStueber) [inline src/main.tsx:44]: I suppose that this comma here is wrong.

**Review by TorstenStueber — CHANGES_REQUESTED** [review]
Great PR, so many small improvements! Thanks.

Only requested change is to remove the extra comma in the JSX code in the main app file.

File: src/components/Layout/Nav/index.tsx
Comment (ebma) [inline src/components/Layout/Nav/index.tsx:1]: Nit: It's unrelated to this PR but `yarn lint` says
```
/Users/marcel/Documents/portal/src/components/Layout/Nav/index.tsx
  1:16  warning  'useEffect' is defined but never used. Allowed unused vars must match /^_/u  @typescript-eslint/no-unused-vars
  5:23  warning  'LinkItem' is defined but never used. Allowed unused vars must match /^_/u   @typescript-eslint/no-unused-vars

```

File: src/components/Form/From/NumericInput/helpers.ts (hunk 1 above)
Comment (ebma) [reply src/components/Form/From/NumericInput/helpers.ts:1]: It's used in l. 56.

**Review by ebma — APPROVED** [review]
Great changes, thanks a lot for taking the time and doing this refactoring @Sharqiewicz 🙏  Nothing to add to @TorstenStueber's comments.

File: src/components/Form/From/NumericInput/index.tsx (hunk 2 above)
Comment (Sharqiewicz) [reply src/components/Form/From/NumericInput/index.tsx:40]: @TorstenStueber It is because Preact uses raw html attribute names and React not

https://preactjs.com/guide/v10/differences-to-react/#raw-html-attributeproperty-names

File: src/components/nabla/Pools/Backstop/AddLiquidity/index.tsx (hunk 4 above)
Comment (Sharqiewicz) [reply src/components/nabla/Pools/Backstop/AddLiquidity/index.tsx:118]: I think it was ineffective attribute

File: src/main.tsx (hunk 5 above)
Comment (Sharqiewicz) [reply src/main.tsx:44]: Thanks!

**Comment by Sharqiewicz** [comment]
@TorstenStueber @ebma Applied the changes

**Review by ebma — APPROVED** [review]

**Review by TorstenStueber — APPROVED** [review]
👍 Great!
